### ISSUE TITLE
Allow `-long_name`

### ISSUE THREAD

**Issue by apavlenko** [vietjtnguyen/argagg#17]
Is it possible to allow long names with a single dash?
e.g. `-log=errors-only`
obviously, short opt-s grouping should be disabled in this case

**Comment by vietjtnguyen** [comment]
Would the idea be that you can configure the parser just prior to parsing to either parse for short option groups or ignore short option groups and treat multi-character options as long options?

**Comment by apavlenko** [comment]
yes, this is exactly what I mean: the approach of grouping short options is popular but an alternative practice of using long options with a single dash is also popular enough.
so I would recommend the ability to have a flag in c-tor disabling grouping and allowing `-l`, `-log` and `--log` to be the same option.

**Comment by ajonnet** [comment]
XCode puts some default arguments, which breaks the argument parser
"-NSDocumentRevisionsDebugMode"

Either a option to filter them or some sort of compatibility mode would be helpfull

**Comment by vietjtnguyen** [comment]
Merging with #23.

**Comment by vietjtnguyen** [comment]
Actually reopening this after https://github.com/vietjtnguyen/argagg/issues/23#issuecomment-458465334
